docs/flow/modin/experimental/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -0,0 +1,24 @@
+:orphan:
+
+Experimental PandasOnRay Execution
+==================================
+
+Experimental PandasOnRay execution is designed to leave the underlying mechanisms of PandasOnRay
+execution: "Data Transformation", "Data Ingress", "Data Egress" architecturally
+:doc:`unchanged </flow/modin/core/execution/ray/implementations/pandas_on_ray/index>`.
+
+Main differencies between these two executions
+----------------------------------------------
+- new Factory ``PandasOnRayFactory`` -> ``ExperimentalPandasOnRayFactory``
+- new IO class ``PandasOnRayIO`` -> ``ExperimentalPandasOnRayIO``
```

Comment:
Don't we want this to be more elaborated, mentioning actual things that are different, for example, an expanded list of IO operations (`read_*_glob`, `to_csv`, ...)

Reply:
It says here about the difference at the top level.
There is already a list of additional features in experimental mode, but it definitely shouldn't be here.

BTW, `to_csv` isn't experimental :)